packages/python-packages/api-stub-generator/apistub/nodes/_docstring_parser.py

```diff
@@ -0,0 +1,130 @@
+import re
+import inspect
+import logging
+from ._argtype import ArgType
+
+
+logging.getLogger().setLevel(logging.INFO)
+
+# REGEX to parse docstring
+find_arg_regex = "(?<!:):{}\s+([\w]*):(?!:)"
```

Comment:
We should avoid parsing with regex. Figure out some other way to parse the docstrings.

Reply:
Closing this comment since we have to use regex to parse docstring